graphs/frequent_pattern_graph_miner.py

```diff
@@ -0,0 +1,183 @@
+'''
+FP-GraphMiner - A Fast Frequent Pattern Mining Algorithm for Network Graphs
+
+A novel Frequent Pattern Graph Mining algorithm,
+FP-GraphMiner, that compactly represents a set of network graphs as a Frequent Pattern Graph (or FP-Graph).
+This graph can be used to efficiently mine frequent subgraphs including maximal frequent subgraphs and maximum common subgraphs.
+
+URL:https://www.researchgate.net/publication/235255851_FP-GraphMiner_-_A_Fast_Frequent_Pattern_Mining_Algorithm_for_Network_Graphs
+
+'''
+def get_DE(EA):

```

Comment:
What is a DE?  What is an EA?  Why should the reader have to guess?  Why not help your reader by having real variable names that are self documenting?  This is discussed in CONTRIBUTING.md.  Uppercase is reserved for _constants_ in Python.

Python type hints and doctest are also discussed in CONTRIBUTING.md.